open-source/funding.md

```diff
@@ -0,0 +1,68 @@
+(open-source:funding)=
+# Funding for open source
+
+There are many funding opportunities earmarked for open source.
+For example, grants, development contracts, or donations that are targeted for open source work. 2i2c should seek this kind of funding, but we must do so in a way that aligns with our values and those of open communities.[^1]
+
+This page lists some guidelines and principles to follow.
+It is impossible to do everything perfectly each time, but these are best-practices that we should strive for.
+
+[^1]: See [](LINK TO BLOG POST) for a brainstorm on this topic.
+
+## Seek funding that aligns our values and strategy with the community
+
+Funding opportunities should be a clear opportunity to advance the interests of **both** the upstream community and 2i2c.
+Ensure that a funding opportunity has a clear value proposition for the community as well as us.
+Point to specific community goals and documentation that helps us make this case, and ask community leaders if there are any doubts.
+
+## Seek funding that matches our capacity and goals
+
+We should feel comfortable seeking dedicated funding for open source, provided that it meets the following conditions:
+
+- It is targeted for one of our [key communities](key-communities.md).
+- It aligns with our technical strategy OR would provide crucial support to the community.
+- It aligns with our team's skillsets, and would not require us to drop our team capacity in a way that would harm the team.
+- We have a plan for continuing (or discontinuing) employment after the funding ends.
+
+## Be transparent in our mission, strategy, and operations
+
+Regardless of a specific funding opportunity, we should make our mission, strategy, budgets, and operational activities transparent so that others can assess whether our interest in funding is realistic and matched to the goals of communities we work with.
+
+## Get consent before seeking funding
+
+Before seeking funding, tell the appropriate leadership group in a community about our intentions.
+Give an idea for the scope of the funding call, what we hope to do with it, and what kind of activities it would fund (people, events, etc).
+
+## Include others in funding opportunities
+
+If possible, find at least one other community member or organization to join us in applying for funding.
```

Comment:
I think this is both the most important and hardest line! It means that everything you are doing is necessarily collaborative, and (potentially) you have less resources to manage the collaboration.

Really love the thought that has gone into this, hopefully something other non-profits and companies (e.g. curvenote) can adopt/adapt as well.